xbmc/pvr/windows/GUIViewStatePVR.cpp

```diff
@@ -77,8 +77,9 @@ void CGUIViewStateWindowPVRGuide::SaveViewState(void)
 
 CGUIViewStateWindowPVRTimers::CGUIViewStateWindowPVRTimers(const int windowId, const CFileItemList& items) : CGUIViewStatePVR(windowId, items)
 {
-  AddSortMethod(SortByLabel, 551, LABEL_MASKS("%L", "%I", "%L", ""));   // FileName, Size | Foldername, empty
-  AddSortMethod(SortByDate, 552, LABEL_MASKS("%L", "%J", "%L", "%J"));  // FileName, Date | Foldername, Date
+  SortAttribute sortAttributes = (SortAttribute) (SortAttributeIgnoreFolders + SortAttributeIgnoreArticle);
```

Comment:
Won't make any difference in this case, but I see what you mean... Issue addressed.